lnrpc/lightning.proto

```diff
@@ -1105,6 +1117,11 @@ message EstimateFeeRequest {
 
     // Whether unconfirmed outputs should be used as inputs for the transaction.
     bool spend_unconfirmed = 4;
+
+    // CoinSelectionStrategy is the strategy to use for selecting coins
+    // to send bitcoin on-chain to multiple addresses for getting the fee
+    // estimates.
```

Comment:
i think this is a liiiiitle confusing. As a user i dont think id understand what "multiple addresses" has to do with anything. I think you can just say that this is the coin selection strategy used during fee estimation. same for the others. Non blocking though - just my opinion.